moviedb_project/moviedb_project/settings.py

```diff
@@ -20,7 +20,7 @@
 # See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '-77dqp+)4sphq@i50$tfcn_mhoei7si7yp5fxtpm()hy@1z@zj'
+SECRET_KEY = os.environ.get("SECRET_KEY") 
```

Comment:
See the commit note about adding the below to your bin/activate instead of having passwords in your repo
```
export SECRET_KEY="yoursecretkeyhere"
```